.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea
```

Comment:
Please don't commit these files into this .gitignore, they should be globally ignored.